spec/tree_spec.rb

```diff
@@ -2,13 +2,76 @@
 require 'tree'
 
 describe 'Tree' do
-  it 'should be a Class' do
-    expect(described_class.is_a? 'Class').to be_true
+  before(:each) do
+    @tree = AppleTree.new
+  end
+
+  it 'should age' do
+    before_age = @tree.age
```

Comment:
Look into `let` and its advantages.
https://www.relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let